src/lib.rs

```diff
@@ -293,8 +288,8 @@ struct SerializePyObject<'p, 'a> {
 
 impl<'p, 'a> Serialize for SerializePyObject<'p, 'a> {
     fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
-    where
-        S: Serializer,
+        where
```

Comment:
That's what intellij-rust's default formatter does, but unfortunately that's not rustfmt style. You can use rustfmt though through he "format with rustfmt" action. (See [intellij-rust/intellij-rust#431](https://github.com/intellij-rust/intellij-rust/issues/431))

Reply:
Glad that at least one of us reads my code. :laughing: Found the option. Rustfmt all the things.